client.go

```diff
@@ -137,7 +137,7 @@ func (c *Client) authenticate(requestID string) (err error) {
 	return
 }
 
-// ExecuteWithBindings formats a raw Gremlin query, sends it to Gremlin Server, and returns the result.
+// ExecuteWithBindings formats a raw Gremlin query, sends it to Gremlin Server, and returns the result.  NOTE: AWS Neptune does not currently support bindings or rebindings see: https://docs.aws.amazon.com/neptune/latest/userguide/access-graph-gremlin-differences.html
 func (c *Client) ExecuteWithBindings(query string, bindings, rebindings map[string]string) (resp []Response, err error) {
```

Comment:
Given the lack of support, perhaps:
` return nil, errors.New('Bindings not supported')`
or similar